tbmalt/ml/integralfeeds.py

```diff
@@ -38,20 +38,25 @@ def indices(dims, dtype=None, device=None):
 
 
 class IntegralFeed(ABC, Feed):
-    """ABC for Hamiltonian and overlap matrix constructors.
+    r"""ABC for Hamiltonian and overlap matrix constructors.
 
     Subclasses of this abstract base class are responsible for constructing
     the Hamiltonian and overlap matrices.
 
     Arguments:
         device: Device on which the feed object and its contents resides.
         dtype: Floating point dtype used by feed object.
+        vcr: Compression radii in DFTB basis.
```

Comment:
The `tbmalt.ml.integralfeeds.IntegralFeed` is intended to be an abstract class that is method agnostic. Will all feeds have variable compression radii. We will want to move this code into the DFTB location. You can create a second abstract base class there if need be.